### ISSUE TITLE
Handles wp-content folder relocation incorrectly

### ISSUE THREAD

**Issue by OllieJones** [OllieJones/sqlite-object-cache#38]
@thomhsb reported:

> I’m trying to understand why activating the plugin has no effect. I read in another topic that an object-cache.php file was created in wp-content. But that’s not my case, so it must be coming from there.
>
> Could it be because I renamed the wp-content folder via wp-config?

https://wordpress.org/support/topic/no-effect-for-me-2/

**Comment by windhamdavid** [comment]
I might try to help you out with this one given the time. Thanks for you work on this plugin. it's ⭐️⭐️⭐️⭐️⭐️

**Comment by nickchomey** [comment]
Is this really still not working? I can't see why it would be the case... 

In the original issue, they said they set
```
define('WP_CONTENT_DIR', ABSPATH . ‘newfolder‘);
define('WP_CONTENT_URL', ‘https://www.mywebsite.com/newfolder’);
```

this plugin [uses](https://github.com/OllieJones/sqlite-object-cache/blob/5ef973ed0dc5f8e22b45a425cc666c1e2eb3595b/includes/class-sqlite-object-cache.php#L134) WP_CONTENT_DIR to copy the object-cache.php file. 

I suspect user error rather than something wrong with this plugin. Probably best to close this and wait for someone else to report the issue.

**Comment by OllieJones** [comment]
Yeah, I know. I've been hoping for further info from the user, and, frankly, hoping this one will go away.  :-)

Have you tried this? Does it work for you? If so, I love closing tix.

**Comment by nickchomey** [comment]
I just created a new site, changed the wp_content_dir constant, renamed the existing wp-content dir to the new one, installed this plugin. the object-cache.php file shows up where it should. Stats are showing up when i browse a few pages. 

I'd close this

**Comment by OllieJones** [comment]
Thank you @nickchomey.
